packages/widgets/src/lib/components/InferenceWidget/shared/types.ts

```diff
@@ -71,3 +71,8 @@ export interface ImageSegment {
 	imgData?: ImageData;
 	bitmap?: ImageBitmap;
 }
+
+export interface ConversationMessage {
```

Comment:
ChatMessage may be better name?

Reply:
https://github.com/huggingface/huggingface.js/pull/499